packages/frontend/core/src/modules/peek-view/view/peek-view-controls.tsx

```diff
@@ -159,40 +162,51 @@ export const AttachmentPeekViewControls = ({
   className,
   ...rest
 }: DocPeekViewControlsProps) => {
+  const { docId, blockIds: [blockId] = [] } = docRef;
+  const { doc } = useEditor(docId);
```

Comment:
Don't use useEditor here, it's a heavy operation.
If you just want to do it, please find another way, or just don't use it!

Reply:
I'm looking for other ways.